test/unit/services/discordService.test.js

```diff
@@ -6,12 +6,14 @@ const {
   getDiscordMembers,
   removeRoleFromUser,
   setUserDiscordNickname,
+  deleteGroupRoleFromDiscord,
 } = require("../../../services/discordService");
 const { fetchAllUsers } = require("../../../models/users");
 const Sinon = require("sinon");
 const userModel = firestore.collection("users");
 const userDataArray = require("../../fixtures/user/user")();
 const discordMembersArray = require("../../fixtures/discordResponse/discord-response");
+// const { func } = require("joi");
```

Comment:
remove this

Reply:
I commented cuz it was not used but not sure if I can remove.